src/pages/styles.module.css

```
@@ -32,13 +40,33 @@
 }
 
 .featureImage {
-  height: 130px;
-  width: 130px;
+  height: 90px;
+  width: 90px;
 }
 
 .featuresGrid {
   display: grid;
   grid-template-columns: repeat(auto-fit, minmax(360px, 1fr));
-  grid-auto-rows: 330px;
-  gap: 4px;
+  grid-auto-rows: 250px;
+  gap: 2.5rem;
+}
+
+/* new styles */
+
+.padmd {
+  padding: 0 2rem;
+}
+
+.featuresp {
+  line-height: 1.4;
+}
+.featuresh3 {
```

```suggestion
.featuresTitle {
```

See previous comments re: mixing styling approaches & word boundaries. Note also that this would require corresponding changes in the pages/index.js.